### PR TITLE
bug: Add missing import to test templates

### PR DESCRIPTION
**Pull request by ballcoach12**
## What does this PR do?

This PR adds a missing import for `"github.com/testcontainers/testcontainers-go"` to the examples_test.go.tmpl and module_test.go.tmpl files.

## Why is it important?

Without this import, generating a new module will return an error code because the required module is not imported in the test files.

## Related issues

 - Closes [#2816]

## How to test this PR

Generate a new module according to the instructions listed [here](https://golang.testcontainers.org/modules/).  Verify that the list of imports contains `"github.com/testcontainers/testcontainers-go"` in the example  test  file and module test file.

### CODE REVIEW

File: modulegen/_template/examples_test.go.tmpl [hunk 1]
```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 	"fmt"
 	"log"
-
+	"github.com/testcontainers/testcontainers-go"
```
Comment (mdelapenya) [inline modulegen/_template/examples_test.go.tmpl:7]: Include a blank line, as the formatter will add it
```suggestion

	"github.com/testcontainers/testcontainers-go"
```

File: modulegen/_template/module_test.go.tmpl [hunk 2]
```diff
@@ -5,7 +5,7 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/require"
-
+	"github.com/testcontainers/testcontainers-go"
```
Comment (mdelapenya) [inline modulegen/_template/module_test.go.tmpl:8]: Same here:
```suggestion

	"github.com/testcontainers/testcontainers-go"
```

**Review by mdelapenya — COMMENTED** [review]
Nit: the lint will complain if we do not add these blank lines separating imports

Other than that, LGTM

**Comment by mdelapenya** [comment]
@ballcoach12 just to verify this is working, can you paste the output of generating a random module from your workspace? 🙏

**Comment by ballcoach12** [comment]
I'm getting an error back,  but the generated code appears to be correct.

```
Error: >> error synchronizing the dependencies: exit status 2
Usage:
  modulegen new module [flags]

Flags:
  -h, --help           help for module
  -i, --image string   Fully-qualified name of the Docker image to be used by the module
  -n, --name string    Name of the module. Only alphabetical characters are allowed.
  -t, --title string   (Optional) Title of the module name, used to override the name in the case of mixed casing (Mongodb -> MongoDB). Use camel-case when needed. Only alphabetical characters are allowed.

>> error synchronizing the dependencies: exit status 2
exit status 1
```

**Comment by mdelapenya** [comment]
So it means there is something else in the formatting 🤔 Could you please take a deep look into the generated files and the templates ?

**Comment by ballcoach12** [comment]
So just debugging the `make lint` command run that is called from exec.go, I captured the output and it looks like this:

```
out: go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.61.0
golangci-lint run --out-format=colored-line-number --path-prefix=. --verbose -c /.golangci.yml --fix
 err: process_begin: CreateProcess(NULL, dirname g:/BITBUCKET/testcontainers-go-github/testcontainers-go/commons-test.mk, ...) failed.
../../commons-test.mk:1: pipe: No error
level=info msg="golangci-lint has version v1.61.0 built with go1.22.2 from (unknown, modified: ?, mod sum: \"h1:VvbOLaRVWmyxCnUIMTbf1kDsaJbTzH20FAMXTAlQGu8=\") on (unknown)"
Error: can't load config: can't read viper config: open /.golangci.yml: The system cannot find the file specified.
Failed executing command with error: can't load config: can't read viper config: open /.golangci.yml: The system cannot find the file specified.
make: *** [../../commons-test.mk:33: lint] Error 3
```

So maybe a relative path is incorrect?

**Comment by mdelapenya** [comment]
I think [69594b8](https://github.com/testcontainers/testcontainers-go/pull/2828/commits/69594b88c239b81df79bd83fe8f630e0b658943b) should be reverted, as it does not belong to this branch, shouldn't it?

**Comment by ballcoach12** [comment]
Yes.  My mistake there

**Comment by mdelapenya** [comment]
@ballcoach12 thanks for this contribution, but I think #2831 resolved it in a cleaner manner. In any case, thanks for your time and dedication for improving testcontainers-go! 💪

**Comment by mdelapenya** [comment]
@ballcoach12 there are and there will be more issues/bugs to work on, so I encourage you to help the community in making progress with your work. Just let us know if we can be of any help

**Pull request closed without merging**